pkg/operator/gcwatchercontroller/gcwatcher_controller.go

```diff
@@ -183,7 +200,7 @@ func checkFiringAlerts(ctx context.Context, requiredAlertsSet sets.String, prome
 		klog.Warningf("received warnings when querying alerts: %v\n", strings.Join(warnings, ", "))
 	}
 	if err != nil {
-		return fmt.Errorf("error querying alerts: %v", err)
+		return err
```

Comment:
why reverting a more verbose message on what is wrong?

Reply:
`prometheusClient.Query()` is already adding that context.

So I removed it.